src/lucky_migrator/create_table_statement.cr

```diff
@@ -134,12 +143,16 @@ class LuckyMigrator::CreateTableStatement
     "bigint"
   end
 
+  def column_type(type : Bool.class)
+    "boolean"
+  end
+
   def column_type(type : Float.class)
     "decimal"
   end
 
-  def column_type(type : Bool.class)
-    "boolean"
+  def column_type(type : Float.class, precision : Int32, scale : Int32)
```

Comment:
Could we also do two more column types. One for just precision, and another for just scale?

Reply:
Sure. When the user only gives scale should we make the precision = scale? I'm going to make the change with that assumption

Reply:
@mikeeus Actually I just reviewed the documentation on numeric and decimal types and it seems that it would be pretty rare/strange to use precision, but not scale. So I think we can leave it as is (need both precision and scale. Can't do just one or the other)

Reply:
That's what I was thinking too

Reply:
Yeah you nailed it the first time 👍